src/hooks/useFlanksourceUISnippet.tsx

```diff
@@ -0,0 +1,38 @@
+import { useFeatureFlagsContext } from "@flanksource-ui/context/FeatureFlagsContext";
+import { useEffect, useState } from "react";
+
+export function useFlanksourceUISnippet(
+  user?: Record<string, any>,
+  organization?: Record<string, any>
+) {
+  const [isSnippetExecuted, setIsSnippetExecuted] = useState(false);
+
+  const { featureFlags } = useFeatureFlagsContext();
+
+  useEffect(() => {
+    if (isSnippetExecuted) {
+      return;
+    }
+    const snippets = featureFlags?.find((flag) => {
+      return flag.name === "flanksource.ui.snippets";
```

Comment:
if flag.source != local, then log a warning: "UI snippets can only be set at start time. "

Reply:
@moshloop at them moment, because I had added the snippet via DB, for testing purpose, it means it won't work after I add this.